desks-front/src/components/UI/Desk/UserBookings/UserBookingsDisplay.jsx

```diff
@@ -0,0 +1,49 @@
+const DeskBooking = ({ booking }) => {
+    // TODO: [JSSBG-19] this looks a bit shitty i.e. takes up too much space - should be refactored (ideally as a list, Excel list)
+    return (
+        <div className="booking card mb-3">
+            <div className="card-body">
+                <h5 className="card-title">Booking Details</h5>
```

Comment:
I think this is the third time I see this card in code, please encapsulate this in its own component. Though, note that it will be changed once we merge PR by @NikolaDmitrasinovic because he made a different display of such data...